Gemfile.lock

```diff
@@ -117,4 +119,4 @@ DEPENDENCIES
   xcpretty
 
 BUNDLED WITH
-   2.1.4
+   2.2.23
```

Comment:
Bumping the bundler version to the latest and to align with braintree_ios

Reply:
Also a note, users will need to `gem install bundler:2.2.23` to get the new bundler version. I think it's pretty self explanatory though so we don't need to worry about documenting it.